Library/Formula/portmidi.rb

```diff
@@ -36,6 +41,11 @@ def install
     system "make", "-f", "pm_mac/Makefile.osx", "install"
 
     if build.with? "python"
+      resource("Cython").stage do
+        ENV.prepend_create_path "PYTHONPATH", buildpath+"lib/python2.7/site-packages"
+        system "python", "setup.py", "build", "install", "--prefix=#{buildpath}",
```

Comment:
When I tried with `Language::Python.setup_install_args` , **setup.py** stoped with the following **WARNING**.
>WARNING:root:Cython is preferred over pyrex for python3 compatibility.
> Traceback (most recent call last):
>   File "setup.py", line 11, in <module>
>     from Pyrex.Distutils import build_ext
> ImportError: No module named Pyrex.Distutils

I have only Python 2.7 . I don't want `pyrex` for python3.